docs/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "C# SDK"
+slug: "csharp-sdk"
+hidden: false
+metadata: 
+  title: "Optimizely C# SDK"
+createdAt: {}
+updatedAt: "2020-01-22T23:31:33.332Z"
+---
+This reference guide describes how to use the Optimizely C# SDK. Refer to the left hand navigation for all sections in this reference guide.
+
+The Optimizely C# SDK is currently at version 3.2.0.
```

Comment:
Please update this to the latest version, which is 3.4.1

Reply:
done